qase-python-commons/src/qase/commons/models/base.py

```diff
@@ -0,0 +1,8 @@
+import json
+
+from pathlib import PosixPath
+
+
+class Base:
+    def to_json(self) -> str:
+        return json.dumps(self, default=lambda o: o.__dict__ if hasattr(o, '__dict__') else str(o), indent=4)
```

Comment:
Wouldn't it be easier to redefine `__str__` and then log just like

```diff
- self.logger.log_debug(f"Project {code} found: {response.result.to_json()}")
+ self.logger.log_debug(f"Project {code} found: {response.result}")
```

Reply:
Done